config/settings.yml

```diff
@@ -410,6 +411,11 @@ claims_api:
     bucket: ~
   disability_claims_mock_override: false
   schema_dir: config/schemas
+  slack:
+    webhook_url: https://example.com
```

Comment:
*nitpick
Could use the `~` like ya did on `threshold` and `environment_name` for consistency.